src/appmixer/utils/rss/NewFeedItem/component.json

```diff
@@ -6,8 +6,14 @@
     "properties": {
         "schema": {
             "properties": {
-                "url": { "type": "string", "format": "uri" }
-            }
+                "url": {
+                    "type": "string",
+                    "format": "uri"
+                }
+            },
+            "required": [
+                "url"
+            ]
```

Comment:
I think this is a breaking change. Please bump the version in `bundle.json` - version and changelog entry.

Reply:
Is it though? Since this is a trigger connecter, you wouldn't have anything mapped to it from previous connectors and also if you didn't fill the URL, you would get the error mentioned in the issue. You basically couldn't use the connector with empty URL so it was already technically required.

Reply:
Of course I don't understand yet how it works in detail, but if you had an URL filled and I would change it to required, it wouldn't remove the filled URL, right?

Reply:
My bad, it is not a breaking change, just a fix 🤦‍♂️ Sorry for the confusion. This change will definitely not remove any data that has been already entered.

But that still needs a bump (patch version increase) in `bundle.json`. This way customers with the buggy version are offered this fix in their BackOffice.